shark/examples/shark_training/bert_training_load_tf.py

```diff
@@ -0,0 +1,44 @@
+import sys
+from absl import app
+import time
+
+import numpy as np
+import os
+import tempfile
+import tensorflow as tf
+
+from official.nlp.modeling import layers
+from official.nlp.modeling import networks
+from official.nlp.modeling.models import bert_classifier
+
+from shark.shark_trainer import SharkTrainer
+
+tf.random.set_seed(0)
+vocab_size = 100
+NUM_CLASSES = 5
+SEQUENCE_LENGTH = 512
+BATCH_SIZE = 1
+
+if __name__ == "__main__":
+    predict_sample_input = [
+        np.random.randint(5, size=(BATCH_SIZE, SEQUENCE_LENGTH)),
+        np.random.randint(5, size=(BATCH_SIZE, SEQUENCE_LENGTH)),
+        np.random.randint(5, size=(BATCH_SIZE, SEQUENCE_LENGTH))
+    ]
+    sample_input_tensors = [tf.convert_to_tensor(val, dtype=tf.int32) for val in predict_sample_input]
+    num_iter = 10
+    mlir_path = "/tmp/model.mlir"
```

Comment:
can we please default to $PWD/filename.mlir ?  (When running on builders etc we may not have access to /tmp/)

Reply:
done! for now you can actually target where do you want to download the file into and the code will look for it. Of course the generalization of this example is to have a loader that can given a local mlir file you can load it in without downloading ahead of time.